pages/ballot/index.tsx

```diff
@@ -148,6 +44,20 @@ const BallotPage: NextPage<{ mobileNavTitle?: string }> = ({
     (race) => race.office.id
   );
 
+  const [isWelcomeVisible, setIsWelcomeVisible] = useState(
+    localStorage.getItem("voting-guide-welcome-visible") !== "false"
+  );
+
+  const createVotingGuide = useUpsertVotingGuideMutation();
+
+  const handleWelcomeDismissal = () => {
+    setIsWelcomeVisible(false);
+    localStorage.setItem("voting-guide-welcome-visible", "false");
+    createVotingGuide.mutate({ electionId: upcomingElection?.id as string });
```

Comment:
If I'm understanding this correctly, this creates the default voting guide for the user. I see it in the `VotingGuideProvider` as well when one doesn't exist for the user - do we need this here as well or can we wrap the entire thing including the welcome with the provider and let it handle bootstrapping the voting guide?

Reply:
I think you're right that we can move this.  When I first authored this I was thinking that as soon as the user clsoed this welcome message, we would create the voting guide, but then realized there may be ways for them to subvert this and wanted to ensure that the voting guide was created.  Lets pull it and have the VotingGuideProvider handle it.